css/amp.css

```diff
@@ -423,6 +423,9 @@ i-amp-scroll-container.amp-active {
  */
 .i-amphtml-element > [overflow] {
   cursor: pointer;
+  /* position:relative is critical to ensure that [overflow] is displayed
+     above an iframe; z-index is not enough. */
```

Comment:
I assume problem was z-index alone was not creating a stacking context so maybe comment can be updated to `z-index alone is not enough to create a stacking context, element must also be positioned (relatively or absolutely). See https://goo.gl/VzS6TK`